docs/en/binance-loom-ethereum.md

```diff
@@ -0,0 +1,512 @@
+  EventBus.$emit('updateStatus', { currentStatus: 'Transferring to Extdev Gateway.' })
+  await this._transferCoinsToExtdevGateway(amount)
+  EventBus.$emit('updateStatus', { currentStatus: 'Getting withdrawal receipt.' })
+  const data = await this._getWithdrawalReceipt()
+  EventBus.$emit('updateStatus', { currentStatus: 'Withdrawing from Rinkeby Gateway.' })
+  await this._withdrawCoinsFromRinkebyGateway(data)
+}
+```
+
+Again, this function is pretty straightforward. First, it transfers the tokens to the transfer gateway contract on the Loom side:
+
+```js
+async _transferCoinsToExtdevGateway (amount) {
+    const amountInt = amount * 100000000
+    const dAppChainGatewayAddr = this.extdevNetworkConfig['extdev2RinkebyGatewayAddress']
+    const ethAddress = this.accountMapping.ethereum.local.toString()
+    await this.extdevBEP2Contract.methods
+      .approve(dAppChainGatewayAddr, amountInt)
+      .send({ from: ethAddress })
+
+    const timeout = 60 * 1000
+    const ownerMainnetAddr = Address.fromString('eth:' + ethAddress)
+    const loomCoinContractAddress = extdevBEP2Token.networks[this.extdevNetworkConfig['networkId']].address
+    const tokenAddress = Address.fromString(this.extdevNetworkConfig['chainId'] + ':' + loomCoinContractAddress)
+    const mainNetContractAddress = rinkebyBEP2Token.networks[this.rinkebyNetworkConfig['networkId']].address
+    const gatewayContract = this.extdev2RinkebyGatewayContract
+
+    const receiveSignedWithdrawalEvent = new Promise((resolve, reject) => {
+      let timer = setTimeout(
+        () => reject(new Error('Timeout while waiting for withdrawal to be signed')),
+        timeout
+      )
+      const listener = event => {
+        const tokenEthAddress = Address.fromString('eth:' + mainNetContractAddress)
+        if (
+          event.tokenContract.toString() === tokenEthAddress.toString() &&
+          event.tokenOwner.toString() === ownerMainnetAddr.toString()
+        ) {
+          clearTimeout(timer)
+          timer = null
+          gatewayContract.removeAllListeners(Contracts.TransferGateway.EVENT_TOKEN_WITHDRAWAL)
+          console.log('Oracle signed tx ', CryptoUtils.bytesToHexAddr(event.sig))
+          resolve(event)
+        }
+      }
+      gatewayContract.on(Contracts.TransferGateway.EVENT_TOKEN_WITHDRAWAL, listener)
+    })
+    await gatewayContract.withdrawERC20Async(
+      new BN(amountInt, 10),
+      tokenAddress,
+      ownerMainnetAddr
+    )
+    console.log('before receiveSignedWithdrawalEvent')
+    await receiveSignedWithdrawalEvent
+  }
+```
+
+One thing to note. As seen above, calling `await gatewayContract.withdrawERC20Async` creates a pending withdrawal. The pending withdrawal is picked up by the Gateway Oracle, which signs the withdrawal, and notifies the  Gateway. The Gateway emits an event to let the user know they can withdraw their token from the Ethereum Gateway to their Ethereum account by providing the signed withdrawal record.
```

Comment:
The pending withdrawal is signed by the Gateway validators, and event is emitted by the Gateway to notify users a withdrawal has been signed. At that point the user can fetch the signed withdrawal receipt from the Gateway and submit it to the Ethereum Gateway in order to withdraw the token to their Ethereum account.